KalturaMediaServer/src/main/java/com/kaltura/media/server/api/services/KalturaCuePointsService.java

```diff
@@ -27,22 +21,8 @@ public boolean createTimeCuePoints(
 			@WebParam(name = "liveEntryId") final String liveEntryId,
 			@WebParam(name = "interval") final int interval,
 			@WebParam(name = "duration") final int duration
-			)
-	{
-		TimerTask timerTask = new TimerTask() {
-
-			@Override
-			public void run() {
-				ICuePointsManager cuePointsManager = (ICuePointsManager) KalturaServer.getManager(ICuePointsManager.class);
-				cuePointsManager.createPeriodicSyncPoints(liveEntryId, interval, duration);
-			}
-		};
-
-		logger.debug("createTimeCuePoints: liveEntryId [" + liveEntryId + "] interval [" + interval + "] duration [" + duration + "]");
-		Timer timer = new Timer("createPeriodicSyncPoints-" + liveEntryId, true);
-		timer.schedule(timerTask, 0);
-		
-		logger.debug("done");
-		return true;
+			) {
+		logger.error("createTimeCuePoints is depricated");
+		return false;
```

Comment:
Why not remove this service altogether? Either way, deprecated means that the functionality is still available, but should be avoided, which is not the case here